README.md

```diff
@@ -83,7 +84,7 @@ After adding your plugin to the `plugins.json` file, run `npm run docs` to regen
 | **[Encode Env Var Objects - `serverless-plugin-encode-env-var-objects`](https://github.com/yonomi/serverless-plugin-encode-env-var-objects)** <br/> by [yonomi](http://github.com/yonomi) <br/> Serverless plugin to encode any environment variable objects. | ![Github Stars](https://img.shields.io/github/stars/yonomi/serverless-plugin-encode-env-var-objects.svg?label=Stars&style=for-the-badge) <br/> ![NPM Downloads](https://img.shields.io/npm/dt/serverless-plugin-encode-env-var-objects.svg?label=Downloads&style=for-the-badge)|
 | **[Env Generator - `serverless-env-generator`](https://github.com/DieProduktMacher/serverless-env-generator)** <br/> by [DieProduktMacher](http://github.com/DieProduktMacher) <br/> Manage environment variables with YAML and load them with dotenv. Supports variable encryption with KMS, multiple stages and custom profiles. | ![Github Stars](https://img.shields.io/github/stars/DieProduktMacher/serverless-env-generator.svg?label=Stars&style=for-the-badge) <br/> ![NPM Downloads](https://img.shields.io/npm/dt/serverless-env-generator.svg?label=Downloads&style=for-the-badge)|
 | **[Ephemeral - `serverless-ephemeral`](https://github.com/Accenture/serverless-ephemeral)** <br/> by [Accenture](http://github.com/Accenture) <br/> Build and include custom stateless libraries for any language | ![Github Stars](https://img.shields.io/github/stars/Accenture/serverless-ephemeral.svg?label=Stars&style=for-the-badge) <br/> ![NPM Downloads](https://img.shields.io/npm/dt/serverless-ephemeral.svg?label=Downloads&style=for-the-badge)|
-| **[Es Logs - `serverless-es-plugin`](https://github.com/daniel-cottone/serverless-es-logs)** <br/> by [daniel-cottone](http://github.com/daniel-cottone) <br/> A Serverless plugin to transport logs to ElasticSearch | ![Github Stars](https://img.shields.io/github/stars/daniel-cottone/serverless-es-logs.svg?label=Stars&style=for-the-badge) <br/> ![NPM Downloads](https://img.shields.io/npm/dt/serverless-es-plugin.svg?label=Downloads&style=for-the-badge)|
```

Comment:
Why did we change this?

Reply:
Hmm. That's interesting. I just ran `npm run docs` and included the exact output here. Is it possible the json and the markdown have gotten out of sync as part of a different pull request?

I'm assuming I shouldn't make any manual changes to the markdown whatsoever after running the command - please let me know if that is in error.

Reply:
hmm yeah something might have got messed up there. Could you rebase master and try again?